contracts/interfaces/0.8.x/IFilteredMinterV3.sol

```diff
@@ -0,0 +1,223 @@
+        address indexed _coreContract,
+        bytes32 _key,
+        address _value
+    );
+
+    /**
+     * @notice Generic project minter configuration event. Adds value `_value`
+     * to the set of addresses at key `_key` for project `_projectId`.
+     */
+    event ConfigValueAddedToSet(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        bytes32 _key,
+        address _value
+    );
+
+    /**
+     * @notice Generic project minter configuration event. Removes value
+     * `_value` to the set of addresses at key `_key` for project `_projectId`.
+     */
+    event ConfigValueRemovedFromSet(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        bytes32 _key,
+        address _value
+    );
+
+    /// BYTES32
+    /**
+     * @notice Generic project minter configuration event. Sets value of key
+     * `_key` to `_value` for project `_projectId`.
+     */
+    event ConfigValueSet(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        bytes32 _key,
+        bytes32 _value
+    );
+
+    /**
+     * @notice Generic project minter configuration event. Adds value `_value`
+     * to the set of bytes32 at key `_key` for project `_projectId`.
+     */
+    event ConfigValueAddedToSet(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        bytes32 _key,
+        bytes32 _value
+    );
+
+    /**
+     * @notice Generic project minter configuration event. Removes value
+     * `_value` to the set of bytes32 at key `_key` for project `_projectId`.
+     */
+    event ConfigValueRemovedFromSet(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        bytes32 _key,
+        bytes32 _value
+    );
+    ///////////////////////////
+    /**
+     * @notice Price per token in wei updated for project `_projectId` to
+     * `_pricePerTokenInWei`.
+     */
+    event PricePerTokenInWeiUpdated(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        uint256 indexed _pricePerTokenInWei
+    );
+
+    /**
+     * @notice Currency updated for project `_projectId` to symbol
+     * `_currencySymbol` and address `_currencyAddress`.
+     */
+    event ProjectCurrencyInfoUpdated(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        address indexed _currencyAddress,
+        string _currencySymbol
+    );
+
+    /// togglePurchaseToDisabled updated
+    event PurchaseToDisabledUpdated(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        bool _purchaseToDisabled
+    );
+
+    /**
+     * @notice Local max invocations for project `_projectId`, tied to core contract `_coreContractAddress`,
+     * updated to `_maxInvocations`.
+     */
+    event ProjectMaxInvocationsLimitUpdated(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        uint256 _maxInvocations
+    );
```

Comment:
Related to my comment in the other file, this actually does look to be defined twice IIUC.